model/scheme.go

```diff
@@ -5,18 +5,22 @@ package model
 
 import (
 	"encoding/json"
+	"fmt"
 	"io"
+	"regexp"
 )
 
 const (
-	SCHEME_NAME_MAX_LENGTH        = 64
-	SCHEME_DESCRIPTION_MAX_LENGTH = 1024
-	SCHEME_SCOPE_TEAM             = "team"
-	SCHEME_SCOPE_CHANNEL          = "channel"
+	SCHEME_DISPLAY_NAME_MAX_LENGTH = 64
```

Comment:
@grundleborg I left `(*Scheme).DisplayName` at max 64 characters (which is what `(*Scheme).Name` was). Should it actually be 128 characters like `(*Role).DisplayName`?

Reply:
Yeah feel free to change it to 128. I don't recall any reason why it's shorter for this one.